### DISCUSSION TITLE
请问author为什么只能显示名字的前两个字啊？在线等，挺急的

### DISCUSSION THREAD

**Discussion by jpDeng111** [sjtug/SJTUThesis#599] [提问] [answered]
**问题描述**:
如题，挺急的

**系统信息**:

- OS版本(带版本号): macOS/Windows/Linux
- TeX 发行版(带版本号): TeX Live/MiKTeX/MacTeX

**日志信息**:

**Comment by skyzh** [comment]
麻烦提供相关的代码？

**Comment by jpDeng111** [comment]
啊谢谢！我马上找一下

**Comment by jpDeng111** [comment]
\sjtusetup{
info = {
title = {上海交通大学学位论文\LaTeX{} 模板示例文档},
title* = {A Sample Document for \LaTeX-based SJTU Thesis Template},
author = {某\quad{}某},
author* = {Mo Mo},
},
style = {
header-logo-color = red,
},
name = {
publications = {攻读学位期间完成的论文},
},
}

这个是交大模板里面的代码

**Comment by jpDeng111** [comment]
作者
259 author = { name = author@zh },
260 author* = { name = author@en },

这个是实现细节里面的代码

**Comment by skyzh** [comment, marked as answer]
这里的姓名确实只有两个字。如果有多个字去掉 \quad 按需填写即可。

**Comment by jpDeng111** [comment]
谢谢！我试试

**Comment by jpDeng111** [comment]
解决了，真的很谢谢你！：）
